doc/02-Installation.md

```diff
@@ -57,8 +92,7 @@ logonmodule="LogonIcingaweb2"
 htmlcgi = "/icingaweb2"
 
 [defaults]
-; This selects the backend_ndomy_1 defined below
-backend = "ndomy_1"
+backend = "ido"
```

Comment:
I would not change this backend because the IDO/NDO backend is already included in NagVis.